gnmi_cli_py/py_gnmicli.py

```diff
@@ -197,6 +197,11 @@ def _parse_path(p_names):
     XpathError: Unabled to parse the xpath provided.
   """
   gnmi_elems = []
+  gnmi_origin = None
+  if p_names and ":" in p_names[0]:
+    res = p_names[0].split(":", 1)
```

Comment:
>":" [](http://example.com/codeflow?start=27&length=3)

If it your invention or gnmi protocol?

Reply:
This is not gnmi protocol.
Google gnxi was using the same solution: https://github.com/google/gnxi/commit/89f51f0ce1e24323174626e8d236623b2d12f09b
Now they use prefix to support origin:
https://github.com/google/gnxi/pull/295
Please let me know which one is better.

Reply:
According to gnmi spec: In the case that a prefix is specified, it MUST specify any required origin
Now I use prefix to support origin